__mocks__/patternlab-node.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+//const patternlab = jest.genMockFromModule('patternlab-node');
```

Comment:
Consider cleaning up this line, if there aren't plans to use it in the future

Reply:
ADDRESSED